### DISCUSSION TITLE
wWETH did not transfer through the wormhole

### DISCUSSION THREAD

**Discussion by rubysfuturellc** [wormhole-foundation/wormhole#434] [Problems / bugs] [answered]
Hey dev team, would love a little help with this. I, like many people,  did not do enough research and transferred wETH through the wormhole only to get wWETH which is essentially useless on Solana right now with no pools to swap it for something else. I tried transferring it back and it left my Solana wallet but has not appeared in my Metamask wallet.

Transaction ID: https://explorer.solana.com/tx/5ESKMchvwYPREuAd8SQ6Dv9yoXVTWQA16MFaw37SXPRnUKiJC3G7c42ERUZ8WwoZi6i7Bi7rKB4GzyWtQxTuST8V

Thanks in advnace for your help.

Cheers!

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest (for reference only): 750eb474666136cc46b42ce69b747470c3c8071c66c1e974810c9e36b1359f8f

Signed VAA bytes to submit to the contract:

> 0x01000000010d00c1dbf3354231dadca023a05a0b676fc6d3218afbeb9b55c8aba85f9440245eb214623a769dfbe2a924dab564550d819b890a66fc4a3ba86f54e1f5d16770bf660001b9f7a069e92f6b0c5991932c90401479f4641cc6ad7e638ab98b170c331732de1b4b47fe6a01ed01df41b061a0a0f38829431a236894168143804e218f6e67c701024e52835b9d2fc6b62fe9b946c557e6580cf29a4221d2325204bd86288b4e7da4266b73ee71aa85ed1b0b365b512bece97b884cc2692c6f0092b944f378a2ed0c0103e31047e329d211b6b3c3b2926215040604d358c1a7ebac038f192cf5c0d64ac900e414f5a9d859a2592d6ce9b9345bc46397ceeadd8fc6771cdda9481ca7b8480104386cb3dcba354a52ec3bd2f7d9d2655f5dc26cf0d2d34ef2315309e08b0e94594b7af7ae84b2652310a89572d4be2033095e10414b35cf77b3e20348be8a597200058d8971aaca3750f669908f345e46504da41a4f8b26450aa29bda9818ceff5070101749ca5e400f425cf21f2a7a6c19f90cb36493d78fff5dd9d8e762a961f2660107466dfa787e1a96d03569bd28f6474d9a455ccc0cffa479d056301e7021dfd67f5e7a2462da7b1e7ad5fb61bdc48d2cf028458956e90dd0061bc2efca0d14764601083256a2a58bf3b77f3e12beb44de29031409493b89670acb6e15f45facd0872ff1e5424d4046fe3a0fb1f09482742721b388f8446c17f5400eb9d3a4c09f979cd000aa7dcdff6858026f06d3e41add8f311c711067276ad34a4c871865220c113e78b5b2d24049434cb03128eb9c6bb58e8505f3dddaeec2388d4aa7e9a19ab4cc250010b98216382259ff6970e3e78a04feb043e6fe2151a0cd1e935a6f59d273f26f0013ecd50422da0248a448aafcb5027fb57d508c56d06ed0848cc34cae0afbefd18010dda60bace44bcea9e9a929028e076a0f3498fa88d948736c5655f173ea12647950106965130e66db160fcfdaf9b8a76d40d7ed7c1c6fe4485754e5c9aba089e4a01102723fc234872d030e3cc6575e59b31db7731e605fa82cc351fb5d0037b37033b40b6dacaee39604c7b3c5d452e48a35f1a7acc05c28caf59b0c575f96e4893c50012c0f79defd81ba3dd5d39eaaaedab88b6f5e5a85b575be69f2cc986847d88d104796b700a577b1d7554b4f1f01fdc62c90eb190d6b4d86e273534f015025879b400613ffcf11000015f4f0102d4b27739b2f3bc7e8abfd8514e92f018d3854613419c6102bf44afd3e0be09d9000000000000000000000000ff2f381c79a7a63de39c8dc77a5869a1ab2fcef102000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc209000000000000000000000000000000000000000000000000000000003b9aca00

**Reply by rubysfuturellc** [reply]
Thank you. That seems to have worked.
